week-1/exercises/decimal.js

```diff
@@ -1,4 +1,12 @@
 const num = 58.4567;
+const wholeNumberPart=Math.floor(num);
+console.log(wholeNumberPart);
+
+const decimalPart= num-Math.floor(num) ;
```

Comment:
you've already assigned Math.floor(num) to a const, so you could have reused that instead of calling Math.floor again, e.g. const decimalPart = num - wholeNumberPart